### DISCUSSION TITLE
Katana Not Able to Find Other Links Issue

### DISCUSSION THREAD

**Discussion by bryanmatthew21** [projectdiscovery/katana#735] [Q&A]
Hello everyone, I have an issue with my Katana. After installing it, it worked well. Then, a few hours later, when trying again I was unable to scan the same domain that used to work, and the other domains as well. I only got one URL as the output, which is the one I typed as the argument to Katana.

For example: katana -u https://google.com will return the output https://google.com
![katana_error](https://github.com/projectdiscovery/katana/assets/24372041/88391811-eedd-4038-b109-97bdd570b46c)

I have tried using other domains, and adding other options like -jc, -d, etc.. but still did not work.

If anyone has some suggestion on how to fix it, it'd be greatly appreciated.
Thankyou so much,
Bryan.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
